python/llm/src/bigdl/llm/utils/isa_checker.py

```diff
@@ -32,7 +32,7 @@ def _avx_vnni(self, cpuid):
             b"\x0f\xa2",               # cpuid
             b"\xC3"                    # ret
         )
-        return ((0x10) & eax) != 0
+        return ((0x10) & int(eax)) != 0
```

Comment:
To fix this error when generating the doc: `unsupported operand type(s) for &: 'int' and '_run_asm'`